Index: /playwright.config.ts
===================================================================
--- /playwright.config.ts
+++ /playwright.config.ts
@@ -3,11 +3,11 @@
 /**
  * Read environment variables from file.
  * https://github.com/motdotla/dotenv
  */
-// import dotenv from 'dotenv';
-// import path from 'path';
-// dotenv.config({ path: path.resolve(__dirname, '.env') });
+import dotenv from "dotenv";
+import path from "path";
+dotenv.config({ path: path.resolve(__dirname, ".env") });
 
 /**
  * See https://playwright.dev/docs/test-configuration.
  */
@@ -21,10 +21,15 @@
   use: {
     trace: "on-first-retry",
   },
   projects: [
+    { name: "setup", testMatch: /login\.setup.ts/ },
     {
       name: "chromium",
-      use: { ...devices["Desktop Chrome"] },
+      use: {
+        ...devices["Desktop Chrome"],
+        storageState: "playwright/.auth/user.json",
+      },
+      dependencies: ["setup"],
     },
   ],
 });
